packages/react-composites/src/composites/ChatComposite/ImageUpload/ImageUploadUtils.tsx

```diff
@@ -124,7 +124,10 @@ const inlineImageUploadHandler = async (
 
     try {
       const response = await adapter.uploadImage(image, task.metadata?.name);
-      uploadTask.notifyUploadCompleted(response.id, task.metadata.url || '');
+      // Use response id as the image src because we need to keep the original image id as a reference to find the image.
+      // Also the html content we send to ChatSDK does not need image src,
+      // it only need the response id to match the uploaded image, url is not needed.
+      uploadTask.notifyUploadCompleted(task.metadata.id, response.id);
```

Comment:
Should this method still name the parameter with url then?

Reply:
@palatter notifyUploadCompleted is a beta API shared with file upload. For image upload, we don't need it to be public. Do you think we should create a local version of it just for image upload?

Reply:
I think we also use this api elsewhere now that looking closer. And the other use is with url. So best to keep just the comment.